packages/components/app/styles/components/table.scss

```diff
@@ -0,0 +1,80 @@
+//
+// TABLE
+//
+// properties within each class are sorted alphabetically
+//
+.hds-table {
+  border-collapse: collapse;
+  border-radius: 6px;
+  border-spacing: 0;
+  border: 1px solid var(--token-color-border-primary);
+  font-family:var(--token-typography-font-stack-text);
+  font-size: var(--token-typography-body-200-font-size);
+  line-height: var(--token-typography-body-200-line-height);
+  width: 100%;
+}
+.hds-table__thead {
+  border-bottom: 1px solid var(--token-color-border-primary);
+  
+  th {
+    background-color: var(--token-color-surface-strong);
+    color: var(--token-color-foreground-strong);
+    font-weight: var(--token-typography-font-weight-semibold);
+    padding: 8px 16px;
+    text-align: left;
+
+    &.hds-table__th--sortable {
+      padding: 0; //we're taking it out here because we need it on the interactive content instead
+    }
+
+    &.text-right { // for num support
+      text-align: right;
+    }
+    a { // might refactor this to a button TBH
+      align-items: center;
+      border: 3px solid transparent;
+      color: var(--token-color-foreground-strong);
+      display: flex;
+      padding: 8px 16px;
```

Comment:
Suggestion/Nit: Would you want to create a sass variable or something to avoid repeating these padding values?

Reply:
I think once they are settled then that seems reasonable, my first goal was to get them into a shape that mostly resembled what was in the figma file (which is still changing, so I'll probably wait to make additional style changes for right now) 👍

Reply:
Once the HTML/CSS code is finalized/settled, we can decide if (and which) compont-specific design tokens to introduce (similar to what we did for the Form controls, that paid off beyond expectations).